src/GDGUkraine/rest_controller.py

```diff
@@ -511,6 +511,23 @@ def generate_invites(self, id):
             raise HTTPError(500, "Cannot save generated invites") from e
         return {"ok": True}
 
+    @cherrypy.tools.json_out()
+    @cherrypy.tools.authorize()
+    def record_visit(self, id):
+        '''POST /api/events/:id/check-in'''
+        req = cherrypy.request
+        orm_session = req.orm_session
+        reg_id = int(id)
+        reg_data = api.get_event_registration_by_id(orm_session, reg_id)
+        if not reg_data:
+            raise HTTPError(400,
```

Comment:
Why `400`?
As I got the code right, `reg_data` will be `None` only in case we have no requested registration in database.
It will be a little bit more comprehensive if we return `404 Not Found` error code in this case, won't it?

P.S. I do not insist on changing error code. I just want to understand why `400 Bad request`?

P.P.S. @phiNumber I'm glad you joined this project ;)